charts/lfx-platform/values.yaml

```diff
@@ -11,6 +11,17 @@ lfx:
     registry: linuxfoundation
     pullPolicy: IfNotPresent
 
+fga-operator:
+  enabled: true
+  # Non-chart value
+  store: "lfx-core"
+  controllerManager:
+    openFgaUrlEnvVar: 'http://lfx-platform-openfga.lfx.svc.cluster.local:8080'
```

Comment:
can this be just `http://lfx-platform-openfga:8080` -- it'll still have the helm deployment name hardcoded, but at least it won't hardcode the namespace

Reply:
Yep, updated.